app-backend/project/repositories

```diff
@@ -0,0 +1,4 @@
+[repositories]
+  local
+  ivy-proxy-releases: http://host.docker.internal:8081/repository/ivy-public/, [organization]/[module]/(scala_[scalaVersion]/)(sbt_[sbtVersion]/)[revision]/[type]s/[artifact](-[classifier]).[ext]
```

Comment:
I wasn't able to get this to work in vagrant because of https://github.com/docker/for-linux/issues/264. Instead, I had to make sure the container I was using for `sbt` commands was connected to the same network as the `nexus` container, then I could just set the proxy host to `http://nexus:8081`.